packages/discovery-react-components/src/components/CIDocument/components/CIDocument/CIDocument.tsx

```diff
@@ -16,49 +16,48 @@ import get from 'lodash/get';
 import flattenDeep from 'lodash/flattenDeep';
 import difference from 'lodash/difference';
 import isEqual from 'lodash/isEqual';
-import DetailsPane from '../DetailsPane/DetailsPane';
-import FilterPanel from '../FilterPanel/FilterPanel';
-import MetadataPane from '../MetadataPane/MetadataPane';
-import NavigationToolbar from '../NavigationToolbar/NavigationToolbar';
-import CIDocumentContent from '../CIDocumentContent/CIDocumentContent';
-import processDoc from '../../../../utils/document/processDoc';
-import { getEnrichmentName } from '../../utils/enrichmentUtils';
-import { getDetailsFromItem, getDetailsFromMetadata } from '../../utils/details';
-import { getFilterHelper, ProcessFilter } from '../../utils/filterHelper';
-import { isFilterEmpty, updateFilter } from '../../utils/filterUtils';
-import { getId, findElement, findElementIndex } from '../../../../utils/document/idUtils';
+import DetailsPane from '@CIDocument/components/DetailsPane/DetailsPane';
```

Comment:
how do we feel about `components/CIDocument` instead of `@CIDocument`?

Reply:
I'm open to using that convention instead if we don't feel we need a prefix.